ksql-rest-app/src/main/java/io/confluent/ksql/rest/util/EntityUtil.java

```diff
@@ -63,20 +67,40 @@ private static SchemaInfo buildSchemaEntity(final Schema schema) {
                 .map(
                     f -> new FieldInfo(f.name(), buildSchemaEntity(f.schema())))
                 .collect(Collectors.toList()),
+            null,
             null
         );
       default:
-        return new SchemaInfo(getSchemaTypeString(schema), null, null);
+        // The list of parameters in the schema must be set in the order they will be displayed.
+        // i.e DECIMAL(precision, value)
+        final List<String> parameters = (schema.parameters() == null) ? null :
+            schema.parameters().values().stream().collect(Collectors.toList());
```

Comment:
Can be simplified to:
`new ArrayList<>(schema.parameters().values());`

Reply:
Prefer `ImmutableList.copyOf(schema.parameters().values())`

Reply:
I removed this code and did what Andy recommended above instead.